lib/linguist/languages.yml

```diff
@@ -2615,6 +2615,18 @@ KRL:
   tm_scope: none
   ace_mode: text
   language_id: 186
+Kaitai Struct:
+  type: programming
+  aliases:
+  - ksy
+  ace_mode: yaml
+  codemirror_mode: yaml
+  codemirror_mime_type: text/x-yaml
+  color: "#773b37"
+  extensions:
+  - ".ksy"
```

Comment:
For other reviewers: I counted 284 repositories by 239 users with the search query [`extension:ksy meta OR seq OR types`](https://github.com/search?p=1&q=extension%3Aksy+meta+OR+seq+OR+types&type=Code).

Reply:
Thanks, that saves me 45 minutes staring at the screen waiting for Harvester to finish.